### ISSUE TITLE
[Retail Bug Report] Pet Battle UI non-existent

### ISSUE THREAD

**Issue by Lehiwa** [tukui-org/ElvUI#664]
### What is the issue you are having?

I had just updated to 13.09, logged in, entered a pet battle and pet battle interface does not appear. 

![Pet battle error](https://user-images.githubusercontent.com/49692878/205476879-f1623723-d933-4561-96a7-036c7f9f8d2b.jpg)

### What is the expected behavior?

To have a pet battle interface.

### What actually happened?

No interface to interact with pets in battle, or surrender from battle, or to do anything other than exit game.

### Errors

```shell
LUA 1:
Message: MultiBarRight:SetScale(): Scale must be > 0
Time: Sat Dec  3 21:50:26 2022
Count: 1
Stack: MultiBarRight:SetScale(): Scale must be > 0
[string "=[C]"]: ?
[string "=[C]"]: in function `SetScaleBase'
[string "@Interface/FrameXML/EditModeSystemTemplates.lua"]:48: in function `SetScale'
[string "@Interface/FrameXML/EditModeManager.lua"]:584: in function `UpdateRightActionBarPositions'
[string "@Interface/FrameXML/EditModeManager.lua"]:556: in function `UpdateActionBarLayout'
[string "@Interface/FrameXML/EditModeSystemTemplates.lua"]:676: in function `UpdateGridLayout'
[string "@Interface/FrameXML/ActionBar.lua"]:350: in function `UpdateVisibility'
[string "@Interface/FrameXML/ActionBar.lua"]:317: in function `HideOverride'
[string "@Interface/FrameXML/ActionBar.lua"]:302: in function `SetShown'
[string "@Interface/FrameXML/MultiActionBars.lua"]:35: in function <Interface/FrameXML/MultiActionBars.lua:30>
[string "@Interface/FrameXML/MultiActionBars.lua"]:43: in function `MultiActionBar_Update'
[string "@Interface/FrameXML/UIParent.lua"]:1672: in function <Interface/FrameXML/UIParent.lua:1362>

Locals: 

LUA 2:
Message: MultiBarRight:SetScale(): Scale must be > 0
Time: Sat Dec  3 21:50:26 2022
Count: 1
Stack: MultiBarRight:SetScale(): Scale must be > 0
[string "=[C]"]: ?
[string "=[C]"]: in function `SetScaleBase'
[string "@Interface/FrameXML/EditModeSystemTemplates.lua"]:48: in function `SetScale'
[string "@Interface/FrameXML/EditModeManager.lua"]:584: in function `UpdateRightActionBarPositions'
[string "@Interface/FrameXML/EditModeManager.lua"]:556: in function `UpdateActionBarLayout'
[string "@Interface/FrameXML/EditModeSystemTemplates.lua"]:676: in function `UpdateGridLayout'
[string "@Interface/FrameXML/ActionBar.lua"]:350: in function `UpdateVisibility'
[string "@Interface/FrameXML/ActionBar.lua"]:310: in function `ShowOverride'
[string "@Interface/FrameXML/ActionBar.lua"]:300: in function `SetShown'
[string "@Interface/FrameXML/MultiActionBars.lua"]:32: in function <Interface/FrameXML/MultiActionBars.lua:30>
[string "@Interface/FrameXML/MultiActionBars.lua"]:43: in function `MultiActionBar_Update'
[string "@Interface/FrameXML/ActionBarController.lua"]:117: in function `UpdateActionBar'
[string "@Interface/FrameXML/ActionBarController.lua"]:125: in function <Interface/FrameXML/ActionBarController.lua:55>
[string "=[C]"]: in function `NotifySettingsLoaded'
[string "@Interface/SharedXML/Settings/Blizzard_SettingsRegistrar.lua"]:19: in function <...ce/SharedXML/Settings/Blizzard_SettingsRegistrar.lua:11>
[string "@Interface/SharedXML/EventUtil.lua"]:21: in function <Interface/SharedXML/EventUtil.lua:15>
[string "=[C]"]: ?
[string "@Interface/SharedXML/CallbackRegistry.lua"]:143: in function <Interface/SharedXML/CallbackRegistry.lua:142>
[string "=[C]"]: ?
[string "@Interface/SharedXML/CallbackRegistry.lua"]:146: in function `TriggerEvent'
[string "@Interface/SharedXML/GlobalCallbackRegistry.lua"]:9: in function <Interface/SharedXML/GlobalCallbackRegistry.lua:8>

Locals: 

LUA 3:
Message: MultiBarRight:SetScale(): Scale must be > 0
Time: Sat Dec  3 21:50:33 2022
Count: 1
Stack: MultiBarRight:SetScale(): Scale must be > 0
[string "=[C]"]: ?
[string "=[C]"]: in function `SetScaleBase'
[string "@Interface/FrameXML/EditModeSystemTemplates.lua"]:48: in function `SetScale'
[string "@Interface/FrameXML/EditModeManager.lua"]:584: in function `UpdateRightActionBarPositions'
[string "@Interface/FrameXML/EditModeManager.lua"]:556: in function `UpdateActionBarLayout'
[string "@Interface/FrameXML/EditModeSystemTemplates.lua"]:676: in function `UpdateGridLayout'
[string "@Interface/FrameXML/ActionBar.lua"]:350: in function `UpdateVisibility'
[string "@Interface/FrameXML/ActionBar.lua"]:317: in function `originalHide'
[string "@Interface/FrameXML/FrameLocks.lua"]:96: in function <Interface/FrameXML/FrameLocks.lua:85>
[string "@Interface/FrameXML/FrameLocks.lua"]:128: in function <Interface/FrameXML/FrameLocks.lua:123>
[string "@Interface/FrameXML/FrameLocks.lua"]:166: in function `AddFrameLock'
[string "@Interface/AddOns/Blizzard_PetBattleUI/Blizzard_PetBattleUI.lua"]:170: in function `PetBattleFrame_Display'
[string "@Interface/AddOns/Blizzard_PetBattleUI/Blizzard_PetBattleUI.lua"]:109: in function <...AddOns/Blizzard_PetBattleUI/Blizzard_PetBattleUI.lua:106>

Locals:
```

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://www.tukui.org/download.php?ui=elvui&changelog).
- [X] I have verfied ElvUI is up to date by checking /estatus.
- [X] I have verfied my issue doesn't exist in the ElvUI Issue Tracker.
- [ ] I have not read step six and I am just clicking all boxes.
- [X] I have verfied my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the three called 'ElvUI', 'ElvUI Libraries', and 'ElvUI Options' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by RAtwood90** [comment]
@Lehiwa 

Troubleshooting Steps (ElvUI)
Do these steps in order.

Type in chat in game /edebug on
Test - Do you still have the issue?
If you still have the issue/error - type in chat in game /estatus
Upload a full, uncropped screenshot showing the issue and the estatus.
If you are finished with debugging, you can enable your AddOns back with: /edebug off

**Comment by kodewdle** [comment]
fixed in 13.10 sorry! :)
